### DISCUSSION TITLE
my distortion tests (with pictures), need advice :)

### DISCUSSION THREAD

**Discussion by embromaster5** [inkstitch/inkstitch#2981] [General]
### Hello :)
First of all, thank you all for creating this beautiful software and keeping it updated, this project is truly a work of wonder. This is my first-ever post on GitHub + english is not my home language so i apologize in advance for any minor inconsistencies :)

**- machine: BROTHER Innov-is V3 (embroidery thread on non-elastic reinforced cotton)**
I´m currently composing (and testing) my first satin design. Lets take a look at the first two tests:
![Média (7)](https://github.com/inkstitch/inkstitch/assets/171574923/e5b204ac-d575-4c4a-846e-9f6bcac5d0b3)
- The design is about the size of a smartphone.
- There is a inner satin + satin outline.
- Consists of 4 symmetrical quadrants of satins (Left/Right + Up/Down) mirrored into a rectangle (so i can later improve 1 quadrant and mirror the other 3).
- As you can see in the first test, the inner parts ( 🟢C ) don´t match with its outline at all, so in the second test - i sew the outline first and all 🟢s afterwards, to minimize the number of operations between 🟢s.
- ( in the 2nd "inversion" i also added 10% pull comp to the inside satins, the first "v2" had 0 pull comp )

### _Question 1:_ **Does this mean the path registration order influences distortion? What would be the best order to match the outline? Should those "vulnerable" parts be close together in the order?**

That brings me to my second question, lets take a look at the svg of "v2" (no pull comp) and the final result:
![image](https://github.com/inkstitch/inkstitch/assets/171574923/5a00823e-fee3-41df-9e4a-397cbfdffcf2)
![Média (9)](https://github.com/inkstitch/inkstitch/assets/171574923/d8828786-9d0b-4d2a-98bd-4726e7211835)
- visually, v2 is good enough, but i as i highlighted with arrows, there is **too much thread in the corners** and the inner spike isn´t very visible

### _Question 2:_ **How can i improve the sharp corners? Should i stop insetting the inside satin rails into the outline and use pull comp?**

Thank you very much for reading through my questions, im very passionate about InkStitch and i want to master all the settings, so im sorry to bother you if my questions were mundane :) (im not a programmer)

HAVE A BLESSED DAY 🍀 ❤️ 

edit: im also excited about the upcoming S-Stitch update, does anyone know when its coming in? thank you 😸

**Comment by kaalleen** [comment]
Hello and welcome to the world of embroidery ;)

> Question 1: Does this mean the path registration order influences distortion? What would be the best order to match the outline? Should those "vulnerable" parts be close together in the order?

Yes, path order definitely has an influence on distortion. You can also anticipate distortion and change your paths accordingly. Also fabric, stabilization and the use of underlays has impact on registration. Learning about embroidery involves a lot of testing. You learn a lot when you use various setups and see what works best for you.

https://inkstitch.org/tutorials/push-pull-compensation/

> Question 2: How can i improve the sharp corners? Should i stop insetting the inside satin rails into the outline and use pull comp?

Have a look at this tutorial here: https://inkstitch.org/tutorials/satin-edges/
It was made before short stitches, but even with short stitches it will make sense to break up sharp edges.

**Reply by embromaster5** [reply]
thank you very much or your insight. i will study the materials and get back to you after more tests :)
